lib/private/docker-credentials.ts

```diff
@@ -106,9 +106,12 @@ export async function fetchDockerLoginCredentials(
   }
 }
 
-export async function obtainEcrCredentials(ecr: IECRClient, logger?: Logger) {
-  if (logger) {
-    logger('Fetching ECR authorization token');
+export async function obtainEcrCredentials(
+  ecr: IECRClient,
+  shellEventPublisher?: ShellEventPublisher
+) {
+  if (shellEventPublisher) {
+    shellEventPublisher('data_stdout', 'Fetching ECR authorization token');
```

Comment:
But this message is not stdout data from a subprocess ?

Reply:
I realized this logging functionality is completely unused today, so I fixed that and also changed it from `logger` across the repo to `eventEmitter` which I think is more accurate

Reply:
I don't understand how your comment responds to mine, sorry.

What I said was, `data_stdout` does not seem an accurate representation of this event message.

Reply:
Yeah realized this was a bit silly as it exists outside the shell so shouldnt be using the shell event publisher so swapped it up to use the `eventEmitter` instead